test/jdk/java/awt/Checkbox/CheckboxCheckerScalingTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.Checkbox;
+import java.awt.Color;
+import java.awt.Frame;
+import java.awt.Rectangle;
+import java.awt.Point;
+import java.awt.Robot;
+import java.awt.image.BufferedImage;
+
+import javax.swing.SwingUtilities;
+
+/*
+ * @test
+ * @key headful
+ * @bug 8233068
+ * @summary Tests checkbox checker on scaling
+ * @requires (os.family == "linux")
+ * @run main CheckboxCheckerScalingTest
+ */
+
+public class CheckboxCheckerScalingTest {
+    private static Frame frame;
+    private static Checkbox checkbox;
+    private static BufferedImage imageAfterChecked;
+    private static volatile boolean checkmarkFound = false;
+
+    public static void main(String[] args) throws Exception {
+        System.setProperty("sun.java2d.uiScale", "2");
+        Robot robot = new Robot();
+        try {
+            SwingUtilities.invokeAndWait(() -> {
```

Comment:
I think `EventQueue.invokeAndWait()` should be used instead of `SwingUtilities.invokeAndWait()` for AWT test.

Reply:
Why?

Reply:
I remember it was advised to use "EventQueue.invokeAndWait() for AWT tests and SwingUtilities.invokeAndWait for swing tests and we should avoid calling swing class from awt.

Reply:
Yes, this is correct. I have seen many AWT tests use `EventQueue.invokeAndWait()`